tests/sentry/sentry_metrics/test_postgres_indexer.py

```diff
@@ -30,3 +31,48 @@ def test_indexer(self):
         # test invalid values
         assert PGStringIndexer().resolve("beep") is None
         assert PGStringIndexer().reverse_resolve(1234) is None
+
+
+class PostgresIndexerV2Test(TestCase):
+    def setUp(self) -> None:
+        self.strings = {"hello", "hey", "hi"}
+        self.indexer = PGStringIndexerV2()
+
+    def tearDown(self) -> None:
+        for obj in StringIndexer.objects.all():
+            key = f"{obj.organization_id}:{obj.string}"
+            indexer_cache.delete(key)
+            indexer_cache.delete(obj.id)
+
+    def test_indexer(self):
+        org_id = self.organization.id
+        org_strings = {org_id: self.strings}
+
+        assert list(
+            indexer_cache.get_many([f"{org_id}:{string}" for string in self.strings]).values()
+        ) == [None, None, None]
+
+        results = PGStringIndexerV2().bulk_record(org_strings=org_strings)
+        obj_ids = list(
+            StringIndexer.objects.filter(string__in=["hello", "hey", "hi"]).values_list(
+                "id", flat=True
+            )
```

Comment:
Please resolve the strings you added from cache. The indexer is supposed to write on the cache. The test can verify this.